Formula/osquery.rb

```diff
@@ -77,8 +78,23 @@ class Osquery < Formula
     sha256 "bf85b2d805f7cd7c4bc0c618c756b02ce618e777b727041ab75197592c4043f2"
   end
 
+  # upstream: 'deps: Update boost to 1.65'
+  # https://github.com/facebook/osquery/pull/3600
+  patch do
+    url "https://github.com/facebook/osquery/commit/70fe4ab194e.patch?full_index=1"
```

Comment:
Since you're already using `boost` 1.65, you shouldn't need to pull in this PR.

Reply:
.... Good point ...